readthedocs/builds/querysets.py

```diff
@@ -141,6 +141,30 @@ def for_reindex(self):
             .distinct()
         )
 
+    def prefetch_subquery(self):
```

Comment:
Why don't we call it `prefetch_latest_build()` instead to keep consistency to what we are doing with `Project` at https://github.com/readthedocs/readthedocs.org/pull/11613/files#diff-05611882195df182851df952b312f031a4314bce04a104e10a09655e570515f3R124?

Reply:
That method shouldn't be called `prefetch_latest_build`, it's doing more than that. Similar here, there are other fields to prefetch.